app/controllers/ConfirmTransferController.scala

```diff
@@ -54,7 +54,7 @@ class ConfirmTransferController @Inject()(val controllerComponents: SecurityComp
       consignmentStatus =>
         val exportTransferStatus = consignmentStatus.flatMap(_.export)
         exportTransferStatus match {
-          case Some("Completed") => Future(Ok(views.html.transferAlreadyCompleted(consignmentId, request.token.name)).uncache())
+          case Some("InProgress") => Future(Ok(views.html.transferAlreadyCompleted(consignmentId, request.token.name)).uncache())
```

Comment:
We've now got three potential status after the user clicks the export/transfer button:

- `InProgress`
- `Completed`
- `Failed`

I think for any of these statuses the user needs to be taken to the completed pages. @techncl What do you think?

Reply:
Ah yes, because the user could possibly come back to the page after the export has completed/failed, right? So I guess line 57 could/should be `case Some("InProgress") | Some("Completed") | Some("Failed")`?

Reply:
@techncl Yes. I think that is correct.

Reply:
Thanks Tom and Nigel. I have updated the code it as per the suggestion.